### PR TITLE
fix: make sure draft admin data is updated properly

### PR DESCRIPTION
**Pull request by nlunets**
Internal incident and manual test revealed that the draft admin data was not maintained properly

As it happens we were not updating the source info properly

### CODE REVIEW

File: packages/fe-mockserver-core/test/unit/v4/services/formSample/FormTemplate.cds [hunk 1]
```diff
@@ -12,7 +12,8 @@ entity FormRoot {
         LastName     : String  @Common.Label :                                'Last Name';
         DateOfBirth  : Date    @Common.Label :                                'Date of Birth';
         EmailAddress : String  @Communication.IsEmailAddress  @Common.Label : 'Email Address';
-        _Elements    : Composition of many SubElements;
+        _Elements    :Association to many SubElements
```
Comment (drktfl) [inline packages/fe-mockserver-core/test/unit/v4/services/formSample/FormTemplate.cds:15]: I wonder why SubElements is still a draft node if it is declared as an association. Maybe a special case in the CDS compiler? Anyway, it has the DraftNode annotation, so it is ok from a test perspective.

File: packages/fe-mockserver-core/test/unit/__testData/RootElement.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [
     {
         "ID": 1,
-        "Prop1": "First Prop",
+        "Prop1": "SomethingElse",
```
Comment (drktfl) [inline packages/fe-mockserver-core/test/unit/__testData/RootElement.json:4]: this is quite annoying, btw.: Running the tests usually results in a change to this file, even if nothing else changed.

Comment (nlunets) [reply packages/fe-mockserver-core/test/unit/__testData/RootElement.json:4]: Yeah :) we should fix that

File: packages/fe-mockserver-core/test/unit/v4/services/formSample/FormTemplate.cds (hunk 1 above)
Comment (nlunets) [reply packages/fe-mockserver-core/test/unit/v4/services/formSample/FormTemplate.cds:15]: Interesting i didn't even think of that, having it as composition remove the partner navigation so i'm lost

**Pull request merged**